conjure-java-core/src/main/java/com/palantir/conjure/java/services/JerseyServiceGenerator.java

```diff
@@ -278,7 +279,10 @@ private ParameterSpec createServiceMethodParameterArg(
                 typeMapper.getClassName(def.getType()), def.getArgName().get());
         if (withAnnotations) {
             getParamTypeAnnotation(def).ifPresent(param::addAnnotation);
-            param.addAnnotations(createMarkers(typeMapper, def.getMarkers()));
+            param.addAnnotations(ImmutableSet.<AnnotationSpec>builder()
+                    .addAll(createMarkers(typeMapper, def.getMarkers()))
```

Comment:
should we warn/error if a user specifys the tag and the marker?

Reply:
That's probably for the best, otherwise we'd see the value twice in request logs in the bad case.